django/db/backends/postgresql_psycopg2/base.py

```diff
@@ -231,6 +235,23 @@ def is_usable(self):
             return True
 
     @cached_property
+    def _nodb_connection(self):
+        nodb_connection = super(DatabaseWrapper, self)._nodb_connection
+        try:
+            nodb_connection.ensure_connection()
+        except (DatabaseError, WrappedDatabaseError):
+            warnings.warn(
+                "Unable to create a connection to the 'postgres' database, using "
```

Comment:
Add period. Also would be helpful to mention what drawbacks a user might see due to this. Otherwise I think the warning will leave users wondering why it's a problem.